.circleci/config.yml

```diff
@@ -534,6 +420,8 @@ jobs:
     #parallelism: 8
     steps:
       - setup
+      - skip_unless_changed:
+          paths: ios apps/bare-expo yarn.lock
```

Comment:
I don't think this uses `ios`, rather it tests `apps/bare-expo`, `apps/test-suite`, and `packages`